app/src/main/java/de/xikolo/models/dao/CourseDao.kt

```diff
@@ -17,9 +17,17 @@ class CourseDao(realm: Realm) : BaseDao<Course>(Course::class, realm) {
         defaultSort = "startDate" to Sort.DESCENDING
     }
 
+    fun all() = all("external" to false)
+
+    fun allSortedByTitle(): LiveData<List<Course>> =
+        query()
+            .sort("title", Sort.ASCENDING)
+            .findAllAsync()
+            .asLiveData()
+
     override fun find(id: String?): LiveData<Course> =
         query()
-            .beginGroup()
+                .beginGroup()
```

Comment:
`.beginGroup()` should not be on the same indentation level as `equalTo()` or `or()`